lib/fs.js

```diff
@@ -1867,6 +1867,49 @@ WriteStream.prototype._write = function(data, encoding, cb) {
 };
 
 
+function writev(fd, chunks, position, callback) {
+  function wrapper(err, written) {
+    // Retain a reference to chunks so that they can't be GC'ed too soon.
```

Comment:
Unfortunately I am a culprit of propagating this practice. Best I can tell it originated with e7604b1e. I then furthered what had been done in commits like 7ca77eaf. In retrospect I should have changed this when I introduced `FSReqWrap()` in 819690fd.

Don't worry about changing it for this PR though. There should be a single clean commit that addresses just this issue. I'll put that change on my list.